### ISSUE TITLE
Project not compiling when running "ng serve" due to wrong webpack configuration.

### ISSUE THREAD

**Issue by patelvimal** [single-spa/single-spa-angular#366]
### STEPS TO REPRODUCE

1. Create a new angular 12 application.
2. Integrate single-spa-angular using schematic command "ng add single-sp-angular"
3. Serve the application using command "npm run serve:single-spa:my-todo-app"
4. Compilation failed due to webpack configuration error.
```
[error] ValidationError: Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
 - configuration.devtool should match pattern "^(inline-|hidden-|eval-)?(nosources-)?(cheap-(module-)?)?source-map$".
   BREAKING CHANGE since webpack 5: The devtool option is more strict.
   Please strictly follow the order of the keywords in the pattern.
    at validate (C:\ngx-apps\my-todo-app\node_modules\webpack\node_modules\schema-utils\dist\validate.js:104:11)
    at validateSchema (C:\ngx-apps\my-todo-app\node_modules\webpack\lib\validateSchema.js:78:2)
    at create (C:\ngx-apps\my-todo-app\node_modules\webpack\lib\webpack.js:109:24)
    at webpack (C:\ngx-apps\my-todo-app\node_modules\webpack\lib\webpack.js:151:32)
    at Object.f [as webpackFactory] (C:\ngx-apps\my-todo-app\node_modules\webpack\lib\index.js:55:16)
    at createWebpack (C:\ngx-apps\my-todo-app\node_modules\@angular-devkit\build-webpack\src\webpack-dev-server\index.js:21:36)
    at Object.runWebpackDevServer (C:\ngx-apps\my-todo-app\node_modules\@angular-devkit\build-webpack\src\webpack-dev-server\index.js:51:12)
    at SwitchMapSubscriber.project (C:\ngx-apps\my-todo-app\node_modules\@angular-devkit\build-angular\src\dev-server\index.js:226:32)
    at SwitchMapSubscriber._next (C:\ngx-apps\my-todo-app\node_modules\rxjs\internal\operators\switchMap.js:47:27)
    at SwitchMapSubscriber.Subscriber.next (C:\ngx-apps\my-todo-app\node_modules\rxjs\internal\Subscriber.js:66:18)
```

### Expected Behavior
Project should compile successfully.
### Actual Behavior
Compilation failed

**Comment by joeldenning** [comment]
Fix at #369

**Comment by joeldenning** [comment]
Oops I didn't see you had fixed it in https://github.com/single-spa/single-spa-angular/pull/367. Closed #369 in favor of #367

**Comment by joeldenning** [comment]
Fix released in https://github.com/single-spa/single-spa-angular/releases/tag/v5.0.1
